src/replica/mutation.cpp

```diff
@@ -135,6 +137,9 @@ void mutation::copy_from(mutation_ptr &old)
 
 void mutation::add_client_request(task_code code, dsn::message_ex *request)
 {
+    if (request != nullptr) {
+        tracer->add_point(fmt::format("mutation::add_client_request({})", request->header->id));
```

Comment:
should we add point if `request == nullptr`?

Reply:
Yeah, such as:
https://github.com/XiaoMi/rdsn/blob/40d86f7f7aa51ba874d9cc91cb7e267f18b2bd11/src/replica/bulk_load/replica_bulk_loader.cpp#L536